### PR TITLE
Error message for unmatched collobarators emails issue #953

### PR DESCRIPTION
**Pull request by MontrealSergiy**
Note I not sure is there security risk. While neurohub already make it easy to check is user email in the system, the bugfix makes it easier further. May be not big deal may be something should be done to alleviate, like allow users to keep their emails confidential, not sure.

### CODE REVIEW

**Comment by prioux** [comment]
Yep, I plan to merge this once I'm done with the main neurohub cbrain merge

**Comment by prioux** [comment]
I have change the merge bass to be 'master' just right now.

File: BrainPortal/app/controllers/nh_invitations_controller.rb [hunk 1]
```diff
@@ -40,12 +40,20 @@ def create #:nodoc:
...
     cb_error "Please specify at least one email address", :redirect => nh_project_path(@nh_project) if user_emails.empty?
-    user_ids        = User.where(:email => user_emails).pluck(:id)
+    users           = User.where(:email => user_emails)
+    user_ids        = users.pluck(:id)
+    found_emails    = users.pluck(:email)
+    wrong_emails    = user_emails - found_emails
+
+    flash_messages = []
+    if wrong_emails.present?
+      flash_messages << "\nWe are not able to invite user(s) with email(s) #{wrong_emails.join(", ")}. At the moment users are matched by email that they are used to register in NeuroHub. Please confirm with them which email they provided to NeuroHub. "
```
Comment (natacha-beck) [inline BrainPortal/app/controllers/nh_invitations_controller.rb:50]: This PR is fine, I just would like to see some more '\n' in the message, it give a really long message that becomes not readable. 

The message regarding the failing invitation is actually in green (notice) I would like prefer to have it in red (error).

Comment (MontrealSergiy) [reply BrainPortal/app/controllers/nh_invitations_controller.rb]: I would call it pink but resolved. I also added yellow color flash, hope it is fine @prioux @candicecz ?

**Pull request merged**